src/main/g8/src/main/scala/$package$/Main.scala

```diff
@@ -0,0 +1,30 @@
+package $package$
+
+import akka.actor.typed.scaladsl.AbstractBehavior
+import akka.actor.typed.scaladsl.ActorContext
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.management.cluster.bootstrap.ClusterBootstrap
+import akka.management.scaladsl.AkkaManagement
+
+object Main {
+
+  def main(args: Array[String]): Unit = {
+    ActorSystem[Nothing](Main(), "$name;format="word-space,upper-camel"$")
+  }
+
+  def apply(): Behavior[Nothing] = {
+    Behaviors.setup[Nothing](context => new Main(context))
+  }
+}
+
+class Main(context: ActorContext[Nothing]) extends AbstractBehavior[Nothing](context) {
+  val system = context.system
+
+  AkkaManagement(system).start()
+  ClusterBootstrap(system).start()
```

Comment:
Add a shutdown task for Akka Management?

Reply:
That's not needed as far as I know.

Reply:
I though Akka Management HTTP required explicitly invoking `stop()`. Who handles the graceful unbinding of that server, then?

Reply:
It will unbind when the ActorSystem is terminating. More graceful than that is not needed. No business critical requests that must be drained or such.